lib/modules/platform/bitbucket-server/index.ts

```diff
@@ -132,6 +134,40 @@ export async function initPlatform({
     );
   }
 
+  if (!gitAuthor && username) {
+    logger.debug(`Attempting to confirm gitAuthor from username`);
+    const options: HttpOptions = {
+      memCache: false,
+    };
+
+    if (token) {
+      options.token = token;
+    } else {
+      options.username = username;
+      options.password = password;
+    }
+
+    try {
+      const { displayName, emailAddress } = (
+        await bitbucketServerHttp.getJson(
+          `./rest/api/1.0/users/${username}`,
+          options,
+          UserSchema,
+        )
+      ).body;
+
+      platformConfig.gitAuthor = `${displayName} <${emailAddress}>`;
+      platformConfig.gitIgnoredAuthors = [FALLBACK_GIT_AUTHOR];
```

Comment:
Unfortunately I think this might override any configured `gitIgnoredAuthors` by the user. Instead I think we'd be better to configure `renovate@whitesourcesoftware.com` as the default value for `gitIgnoredAuthors`. It's ok to have that for all platforms

Reply:
No, it doesn't override. `gitIgnoredAuthors` from `platformConfig` is merged with `gitIgnoredAuthors` from config. See [lib/modules/platform/index.ts](https://github.com/renovatebot/renovate/pull/29525/files#diff-62ca4fa20e113c91022a1baadcef7616e79f6f500b1d31b3d19b892dfa9f3750).

Reply:
Or do you think it's unnecessary complexity?

Reply:
I don't have a strong opinion about either approach so if you've already written this one then I'm ok with it

Reply:
This shouldn't be set here. It should be set for all platforms or not at all.
At least it needs to be documented!

Reply:
OK, let's add `renovate@whitesourcesoftware.com` and `bot@renovateapp.com` as default `gitIgnoredAuthors`

Reply:
#30313